pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: lints
-version: 2.0.1
+version: 2.1.0
 description: >
   Official Dart lint rules. Defines the 'core' and 'recommended' set of lints
   suggested by the Dart team.
 repository: https://github.com/dart-lang/lints
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: ^3.0.0
```

Comment:
Have the guidelines changed for this? https://dart.dev/tools/pub/pubspec#sdk-constraints still advises to not use the ^ for sdk constraints.

Reply:
Ah, ok.  I forgot that this was still the guidance.  Thanks!

> Do we need to make another release of flutter_lints as well once this lands?

I think we should?

Reply:
>  https://dart.dev/tools/pub/pubspec#sdk-constraints still advises to not use the ^ for sdk constraints.

RE: this guidance, it'd be a good thing to produce a diagnostic for probably?

/fyi @jonasfj @sigurdm

Reply:
We're waiting on updating the guidance until Dart 3 - https://github.com/dart-lang/site-www/issues/4693

Reply:
Cool!

> Let us know if we should change it now or not at all though.

@mit-mit @kevmoo may have opinions?

Reply:
`^` is awesome. We had some weird issue YEARS ago where we wanted to avoid it. Old history....

Reply:
Some of that history on why can be found in https://github.com/dart-lang/pub/issues/2153#issuecomment-1328860726 as well as why it's fine to make the change now :)

Reply:
@parlough I think we could update already now.
We allowed ^ syntax in the sdk constraint already from 2.19 I believe.